composer.json

```diff
@@ -1,5 +1,6 @@
 {
     "name": "php-pm/php-pm",
+    "description": "Process manager for modern PHP applications",
```

Comment:
Should be:

> PHP-PM is a process manager, supercharger and load balancer for PHP applications.

for sake of consistency

Reply:
I agree